lib/extensions/ar_migration.rb

```diff
@@ -55,42 +43,43 @@ def wait_for_remote_region_migration(wait_time = 1)
         print(".")
         restart_subscription
         sleep(wait_time)
-        region.reload
       end
 
       puts("\n")
     end
 
     private
 
-    def wait_for_migration?
-      migrations_column_present? ? !region.migrations_ran&.include?(version) : false
+    def region_number
+      subscription.provider_region
     end
 
-    def migrations_column_present?
-      @migrations_column_present ||= PglogicalMigrationHelper.migrations_column_present?
+    def wait_for_migration?
+      return false unless schema_migrations_ran_class
+      !schema_migrations_ran_class.unscoped.in_region(region_number).where(:version => version).exists?
```

Comment:
Can you just add a comment here to explain the `unscoped` it's a bit odd at first glance and the default scope for the helper class is easy to miss.

Generally I prefer to avoid default scopes but there would be a lot of calls in the local region stuff.

Reply:
Done.